test/cql-pytest/test_describe.py

```diff
@@ -733,6 +733,36 @@ def test_table_options_quoting(cql, test_keyspace):
     finally:
         cql.execute(f"DROP TYPE {test_keyspace}.\"{type_name}\"")
 
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #16317")]), "vnodes"],
+                         indirect=True)
+def test_hide_cdc_table(scylla_only, cql, test_keyspace):
+    cdc_table_suffix = f"_scylla_cdc_log"
+    
+    with new_test_table(cql, test_keyspace, "a int primary key, b int", "WITH cdc = {'enabled': true}") as t:
+        t_name = t.split('.')[1]
+
+        # Check 't_scylla_cdc_log' is hidden in DESC TABLES
+        desc_tables = cql.execute("DESC TABLES")
+        assert t_name + cdc_table_suffix not in [r.name for r in desc_tables]
```

Comment:
This test has a risk that if the cdc_table_suffix is wrong, the test will pass and not test anything :-)
So can you please confirm that this test actually failed before your fix?

Reply:
Perhaps we could explicitly check that the CDC log table with the expected name exists by looking into driver's schema metadata (see [here](https://docs.datastax.com/en/developer/python-driver/3.26/api/cassandra/metadata/#cassandra.metadata.KeyspaceMetadata)).

Reply:
Good idea. But instead of looking at metadata, perhaps even simpler is to try to read something from the cdc table (maybe a silly single-partition select that returns nothing, or something like that) to confirm that the table by that name exists but it's missing in DESC TABLES.

If this test isn't scylla_only already, please make sure it is (since it uses CDC which doesn't exist in this form in Cassandra).